roles/sap_general_preconfigure/handlers/main.yml

```diff
@@ -8,14 +8,6 @@
   when:
     - sap_general_preconfigure_reboot_ok|d(false)
 
-- name: Let the role fail if a reboot is required
```

Comment:
This is all on purpose. We want the role to fail by default if a reboot is required but the role is not configured to automatically reboot the managed node. A reboot requirement should not remain unnoticed by default.
The failure can be switched off by either:
- setting the variable `sap_general_preconfigure_fail_if_reboot_required` to `no` or
- setting the variable `sap_general_preconfigure_reboot_ok` to `yes`.

Reply:
@rainerleber Can you please have a look at this and the other issues in this PR?
@rdvansloten Can you please provide your role variables with regards to the reboot handling?